additional.d.ts

```diff
@@ -19,23 +19,3 @@ declare module 'debug'
 declare module 'mq-polyfill'
 declare module 'nock'
 declare module 'test-utils'
-
-declare namespace JSX {
```

Comment:
I didn't actually need to move this to a new module, but I liked keeping it a little more separate. 🤷